Xcode-Configurations/Base.xcconfig

```diff
@@ -2,15 +2,45 @@ ARCHS[sdk=macosx*] = $(ARCHS_STANDARD_64_BIT)
 ARCHS[sdk=iphonesimulator*] = $(ARCHS_STANDARD)
 ARCHS[sdk=iphoneos*] = $(ARCHS_STANDARD)
 
-MACOSX_DEPLOYMENT_TARGET = 10.8
+MACOSX_DEPLOYMENT_TARGET = 10.10
 IPHONEOS_DEPLOYMENT_TARGET = 8.0
 
 DEBUG_INFORMATION_FORMAT = dwarf-with-dsym
-COMBINE_HIDPI_IMAGES = YES
+TARGETED_DEVICE_FAMILY = 1,2,3,4
+COPY_PHASE_STRIP = NO
```

Comment:
Why do we want this moved to the base config? Don't we want to strip symbols in the release configurations?

Reply:
You can't run stripping on a code-signed binary (like the third-party frameworks we embed) because it breaks its code-signing, so that emits a warning without a change. `COPY_PHASE_STRIP = NO` is now the default on new Xcode projects in all configurations.